backend/fms_core/template_importer/importers/normalization_planning.py

```diff
@@ -269,23 +272,24 @@ def convert_to_numerical_robot_coord(coord_spec, fms_coord) -> int:
             add_library_lines.append((",".join(["SrcBarcode", "SrcName", "SrcWell", "DstName", "DstWell", "DNAVol"]) + "\n").encode())
 
             for output_row_data in output_norm_rows_data:
-                container_src_barcode = output_row_data["Source Container Barcode"]
-                robot_src_barcode = output_row_data["Robot Source Container"]
-                robot_dst_barcode = output_row_data["Robot Destination Container"]
-                robot_src_coord = output_row_data["Robot Source Coord"]
-                robot_dst_coord = output_row_data["Robot Destination Coord"]
-                volume_library = decimal.Decimal(output_row_data["Volume Used (uL)"])
-                volume_diluent = decimal.Decimal(output_row_data["Volume (uL)"]) - volume_library
```

Comment:
So no more "Volume (uL)"?

Reply:
The calculation is made obsolete by passing along the diluent volume (required by the new calculation)